applications/orsi/lib/CMakeLists.txt

```diff
@@ -19,21 +19,21 @@ project(orsi_app_lib CXX)
 find_package(holoscan 0.6 REQUIRED CONFIG
              PATHS "/opt/nvidia/holoscan" "/workspace/holoscan-sdk/install")
 
-add_library(${PROJECT_NAME}
+add_library(orsi_app_lib EXCLUDE_FROM_ALL
```

Comment:
With these changes, do the orsi lib no longer get compiled if we are not building the orsi app?

Reply:
This is correct. This was an issue that it was built by default so building the h264 will build the orsi lib.
Could be put in another MR but good to put here.

Reply:
fine with having this here